src/views/product-tutorials-view/components/product-view-content/components/featured-stack/featured-stack.module.css

```diff
@@ -1,5 +1,6 @@
 .heading {
   composes: hds-typography-display-300 from global;
+  composes: g-offset-scroll-margin-top from global;
```

Comment:
`FeaturedStack` headings appear in the table of contents, so they need `g-offset-scroll-margin-top`. Note that `CollectionsStack` & `TutorialsStack` use `FeaturedStack` as well.